### PR TITLE
Adil/week1

### PR DESCRIPTION
**Pull request by musafir928**

### CODE REVIEW

File: homework-submission/todo-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  });
+};
+
+const App = () => {
+  return (
+    <div>
+      <section className="static-list">
+        <h1 className="static-title">Todos (static list)</h1>
+
+        {staticTodo('Get out of bed', 'Wed Sep 13 2017')}
```
Comment (nick-barth) [inline homework-submission/todo-app/src/App.js]: Why didnt you use the array you declared earlier, to loop through these list items!

File: homework-submission/todo-app/src/component/dynamicComponent/dynamicDescription.jsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+const DynamicDescription = ({ description }) => <div>description: {description}</div>;
```
Comment (nick-barth) [inline homework-submission/todo-app/src/component/dynamicComponent/dynamicDescription.jsx]: I'm not sure these all needed to be their own components, but great job breaking it down!

File: homework-submission/todo-app/src/App.js (hunk 1 above)
Comment (musafir928) [reply homework-submission/todo-app/src/App.js]: Thank you Nick for your review and feedback.

File: homework-submission/todo-app/src/component/dynamicComponent/dynamicDescription.jsx (hunk 2 above)
Comment (musafir928) [reply homework-submission/todo-app/src/component/dynamicComponent/dynamicDescription.jsx]: I tried to make them totally ‘static’.

File: homework-submission/todo-app/src/App.js (hunk 1 above)
Comment (musafir928) [reply homework-submission/todo-app/src/App.js]: I tried to make them totally static.

**Pull request closed without merging**